### ISSUE TITLE
[BUG] `startCase` proper typing

### ISSUE THREAD

**Issue by hakimio** [newdash/newdash#914]
## Describe the bug

Currently `startCase` has the following typing:
```ts
const startCase: (string: any) => string;
```
should be properly typed:
```ts
const startCase: (string: string) => string;
```

## System Environment

* OS: 
* Node Version: 
* This Library Version: 

## To Reproduce

**Provide an example repo is better**

**Comment by Soontao** [comment]
fixed by 5.22.1

**Comment by hakimio** [comment]
Thanks for fixing this so fast.
To avoid bugs like this in the future `isolatedDeclarations` from [TypeScript 5.5](https://devblogs.microsoft.com/typescript/announcing-typescript-5-5-beta/#solution-explicit-types) might be useful.
